configs/kueue/prod/clusterQueue.config.yaml

```diff
@@ -49,7 +58,34 @@ spec:
               borrowingLimit: "13159Gi"
               lendingLimit:   "13159Gi"
             - name: "nvidia.com/gpu"
-              nominalQuota: "18"
+              nominalQuota: "0"
+              borrowingLimit: "0"
+              lendingLimit: "0"
+        - name: "gpu-flavor"
+          # Cluster Hardware Configuration
+          # Does not include
+          #   - skaha.opencadc.org/node-type=service-node (p-nodes)
+          #   - node-role.kubernetes.io/control-plane
+          # Compute Nodes: 40 x [64 cores, 241Gi memory, 2048Gi ephemeral-storage]
+          # GPU Nodes A: 3 x [8 cores, 40Gi memory, 140Gi ephemeral-storage, GRID-V100D-16C]
```

Comment:
The only remaining GPU nodes since the V100s had to be retired are:
- g01, NVIDIA-A100-PCIE-40GB-MIG-3g.20gb
- g02, Tesla-V100-PCIE-32GB

Reply:
Looks like a total of 9 distinct GPU devices.